lib/buildpack/compile/installers/nodejs_installer.rb

```diff
@@ -24,6 +24,10 @@ class NodeJsInstaller < Installer
     CACHE_DIR = '.node'.freeze
     NPM_COMMAND = 'npm'.freeze
 
+    def self.install_order
```

Comment:
Same as the change to the Bower installer, this doesn't have to be changed from the default `install_order` defined in installer.rb.